README.md

```diff
@@ -72,5 +72,5 @@ platforms and build environments are tested:
 * Linux: Ubuntu 14.04, RHEL 6.6 (Makefile, x64)
 * Mac OS X 10.9
 
-The [API documentation](https://bluebrain.github.io/Fivox-0.4/index.html)
+The [API documentation](https://developer.humanbrainproject.eu/docs/Fivox/latest/)
```

Comment:
For the release has to be changed to the actual version, otherwise the release code points always to latest documentation.